src/Sulu/Component/SmartContent/Tests/Unit/ContentTypeTest.php

```diff
@@ -679,7 +673,7 @@ public function testGetContentDataPagedLimit(
             ],
             ['webspaceKey' => 'sulu_io', 'locale' => 'de'],
             $limitResult,
-            $page,
+            $page < 1 ? 1 : ($page > \PHP_INT_MAX ? \PHP_INT_MAX : $page),
```

Comment:
@Prokyonn as now handled in getCurrentPage I think this can be removed.

Reply:
We still need this, because from the request we want to have the invalid values, but on the method call we have to check for the valid values. In the tests the `$page` comes directly from the `pageProvider` which generates also the invalid values

Reply:
Oh sorry thought we were here in the ContentType not in the test 🙈